src/screens/Contact/Contact.tsx

```diff
@@ -154,11 +155,8 @@ const Contact = ({ children }: { children: React.ReactNode }) => {
           </SectionHeader>
           <Paragraph>
             We are a registered 501c3 non-profit. You can help us keep the
-            lights on by making{' '}
-            <ExternalLink href="https://www.gofundme.com/f/qtcuvy-covid-act-now">
-              a 100% tax-deductible donation to our GoFundMe
-            </ExternalLink>
-            , or{' '}
+            lights on by making a 100% tax-deductible donation{' '}
+            <Link to="/donate">here</Link>, or{' '}
```

Comment:
Consider linking "making a 100% tax-deductible donation" instead of "here" to make it more accessible.